web-ui/view/step.ml

```diff
@@ -583,7 +584,7 @@ module Make (M : Git_forge_intf.Forge) = struct
                   ]))
       in
       Printf.sprintf "%s%s"
-        (List.fold_left aux "<table id='steps-table'><tbody>" data)
+        (List.fold_left aux "<table class='steps-table flex'><tbody>" data)
```

Comment:
Can we keep the id here along with the class? We want it to be unique, no?

Reply:
This is actually why I changed it - there are multiple tables that appear according to the streaming mechanism and so `id` is not ok anymore.

Reply:
The break in the column in the logs screenshot above corresponds to a new table being formed.